hearth/lib/hearth.rb

```diff
@@ -27,6 +27,21 @@
 require_relative 'hearth/waiters/waiter'
 require_relative 'hearth/xml'
 
+# Core SDK library
 module Hearth
   VERSION = File.read(File.expand_path('../VERSION', __dir__)).strip
+
+  # @return true if CRT is available
+  def self.use_crt?
```

Comment:
I understand why this is here but I don't know if hearth should have an AWS dependency. I am thinking crc22c should be an SDK core provided class somehow.

Reply:
I dont really see a good way to move only this to AWS.  I might lean towards moving ALL of checksums to AWS (since they will only be used from there) and just leaving the existing content_md5 middleware alone here.

Reply:
That's better